src/openrct2/peep/Staff.cpp

```diff
@@ -2677,3 +2665,19 @@ void Staff::UpdateRideInspected(ride_id_t rideIndex)
             | RIDE_INVALIDATE_RIDE_LIST;
     }
 }
+
+money32 GetStaffWage(StaffType type)
```

Comment:
Do you prefer this over a table? Maybe we should make a StaffTypeDescriptor table same as rides and tracks.

Reply:
Generally yes because it doesn't rely on enum order